app/Http/Controllers/Api/DeviceController.php

```diff
@@ -318,5 +322,69 @@ public function anyList(BcsRequest $request)
 
         return response()->json($ret_val);
     }
-
+    
+    /**
+     * @param \BComeSafe\Http\Requests\SaveDeviceRequest $request
+     *
+     * @return array
+     */
+    public function anyUpdateDevice(SaveDeviceRequest $request)
```

Comment:
Is there any reason why this is `anyUpdateDevice` rather than `postUpdateDevice`?

Reply:
Initially was for testing purposes via Browser. Now changed to POST only.